enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -188,6 +188,9 @@ private void applyQueuedTransactions() throws IOException
             // Check rotation explicitly, since the version of append that we're calling isn't doing that.
             logRotation.rotateLogIfNeeded( LogAppendEvent.NULL );
 
+            // Check kernel health after log rotation
+            kernelHealth.assertHealthy( IOException.class );
```

Comment:
@davidegrohmann @MishaDemianenko On that note: we log that inside `KernelHealth`. That message though is outdated and comes from earlier days where we had a TransactionManager class, refered to as `TM` in there. There are also other variable names refering to that time. We should rather fix that message and maybe make that note there, that the instance probably requires manual intervention and recovery in order to come back online.